GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -80,6 +80,8 @@ typedef void (^GIDDisconnectCompletion)(NSError *_Nullable error);
 /// :nodoc:
 - (instancetype)init NS_UNAVAILABLE;
 
+/// Handles the URL returned from SFSafariViewController for iOS 9 and 10.
```

Comment:
This is accurate but doesn't tell the full story.  The custom URL scheme is used for app flipping in the MDM flow on all iOS versions.  We don't want developers to think they don't need to handle the custom URL scheme if they don't support iOS 9 and 10.

Reply:
Thanks, I have updated the comments.
What is app flipping?

Reply:
The app flipping means the communications between GSI and Device policy app. Clients should always handle custom URL and don't need to know the details. 

We should move this comments into our implementations for records.